README.md

```diff
@@ -53,14 +53,14 @@ sudo git clone https://github.com/andreknieriem/photobooth html
 cd /var/www/html
 sudo git submodule update --init
 sudo cp config.inc.php my.config.inc.php
-sudo mkdir -p /var/www/html/images
-sudo mkdir -p /var/www/html/keying
-sudo mkdir -p /var/www/html/print
-sudo mkdir -p /var/www/html/qrcodes
-sudo mkdir -p /var/www/html/thumbs
-sudo mkdir -p /var/www/html/tmp
-sudo chown -R pi: /var/www/
-sudo chmod -R 777 /var/www
+sudo mkdir -p /var/www/html/data/images
+sudo mkdir -p /var/www/html/data/keying
+sudo mkdir -p /var/www/html/data/print
+sudo mkdir -p /var/www/html/data/qrcodes
+sudo mkdir -p /var/www/html/data/thumbs
+sudo mkdir -p /var/www/html/data/tmp
+sudo chown -R www-data:www-data /var/www/
+sudo chmod -R u+w /var/www/data
```

Comment:
Must be  `/var/www/html/data`

Reply:
As the label "Outdated" indicates is this not from the final readme.